src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +80,53 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={movieTitle}
+              onChange={handleChangeMovieTitle}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {hasError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              disabled={!movieTitle}
+              className={
+                classNames('button is-light', { 'is-loading finally': loader })
+              }
             >
-              Find a movie
+              {!movieTitle ? 'Search again' : 'Find a movie'}
```

Comment:
it shows 'Search again' by default
![image](https://user-images.githubusercontent.com/86923089/207678083-50afd10d-f09c-454c-bc8f-81b6e3bdbbc6.png)

You should change the condition if you want to show the different text (maybe if the error exists ?)